internal/api/rust/rust.go

```diff
@@ -463,7 +464,9 @@ func (c *RustClient) TrySendMessage(t ct.TestLike, roomID, text string) (eventID
 				// else this will panic on the 2nd call.
```

Comment:
It's just saying "it's important" not that "it will". It's still important, we're just now guarding it much better rather than relying on racey read-modify-writes to `eventID`.